.ci/load/Jenkinsfile

```diff
@@ -164,7 +164,7 @@ pipeline {
         stage('execute test') {
             parallel {
                 stage('Load generation') {
-                    agent { label 'metal' }
+                    agent { label 'linux && metal' }
```

Comment:
Why would this not go into the microbenchmarks pool?

Reply:
```suggestion
                    agent { label 'microbenchmarks-pool' }
```